addon/components/text-field.js

```diff
@@ -27,13 +31,34 @@ export default class TextField extends BaseComponent {
     return nonce++;
   }
 
+  get debounceMs() {
+    let ms = this.args.debounceMs;
+    return ms !== undefined ? this.args.debounceMs : 500;
+  }
+
+
   @action
   updateValue(element, [value, validationMessage, required]) {
     this.value = value;
     this.validationMessage = validationMessage;
     this.required = required;
   }
 
+  @action
+  keyUp(value, evt) {
+    if (this.environment === 'test') {
```

Comment:
One option here is, you can stub `handeInput` in the test instead of adding this into the main codebase.